app/policies/check_in_policy.rb

```diff
@@ -2,7 +2,7 @@
 
 class CheckInPolicy < ApplicationPolicy
   def show?
-    Membership.find_by(project: record.iteration.project, user: user, role: %w[contributor mentor])
+    Membership.find_by(project: record.iteration.project, user: user, role: %w[contributor mentor]) || user.is_admin?
```

Comment:
I wonder if this is an opportunity extract into a helper method on `ApplicationPolicy` as it looks to be used in a number of policies with varying arguments.